paper-tabs.html

```diff
@@ -377,6 +378,39 @@
         }, 10);
       },
 
+      _onIronItemsChanged: function(event) {
```

Comment:
You could just override that function and call it here, like this you don't need to have another listener.
```javascript
/**
 * Overridden from IronMenuBehavior
 */
_onIronItemsChanged: function(event) {
  Polymer.IronMenuBehavior._onIronItemsChanged.apply(this, arguments);
 // ...